pkg/utils/container.go

```diff
@@ -124,18 +123,17 @@ func DeleteTable(logger logger.Logger, container v3io.Container, path, filter st
 
 	responseChan := make(chan *v3io.Response, 1000)
 	commChan := make(chan int, 2)
-	doneChan := respWaitLoop(commChan, responseChan, 10*time.Second)
-	reqMap := map[uint64]bool{}
+	doneChan := make(chan error, 1)
+	go respWaitLoop(commChan, responseChan, doneChan, 10*time.Second)
 
 	i := 0
 	for iter.Next() {
 		name := iter.GetField(config.ObjectNameAttrName).(string)
-		req, err := container.DeleteObject(&v3io.DeleteObjectInput{Path: path + "/" + name}, nil, responseChan)
+		_, err := container.DeleteObject(&v3io.DeleteObjectInput{Path: path + "/" + name}, nil, responseChan)
 		if err != nil {
```

Comment:
Need to check the status code here, so that the worker won't quit and we will continue deleting all remaining items

Reply:
This error is only for sending the request. The response is received asynchronously.